docker/buildkite/docker-compose-local-pinot.yml

```diff
@@ -0,0 +1,207 @@
+      services-network:
+        aliases:
+          - kafka
+
+  # will be deleted later when we get rid of ES usages
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.9.3
+    # version here must be the same as the one in testdata/integration_pinot_cluster
+    ports:
+      - "9200:9200"
+    networks:
+      services-network:
+        aliases:
+          - elasticsearch
+    environment:
+      - discovery.type=single-node
+
+  zookeeper:
+    image: zookeeper:3.5.8
+    restart: always
+    hostname: zookeeper
+    container_name: zookeeper
+    ports:
+      - '2181:2181'
+    environment:
+      - ZOOKEEPER_CLIENT_PORT=2181
+      - ZOOKEEPER_TICK_TIME=2000
+    networks:
+      services-network:
+        aliases:
+          - zookeeper
+
+  pinot-controller:
+    image: apachepinot/pinot:0.12.1
+    command: "StartController -zkAddress zookeeper:2181 -controllerPort 9001"
+    container_name: pinot-controller
+    restart: unless-stopped
+    ports:
+      - "9001:9001"
+    environment:
+      JAVA_OPTS: "-Dplugins.dir=/opt/pinot/plugins -Xms1G -Xmx4G -XX:+UseG1GC -XX:MaxGCPauseMillis=200 -Xloggc:gc-pinot-controller.log"
+    depends_on:
+      - zookeeper
+    networks:
+      services-network:
+        aliases:
+          - pinot-controller
+  pinot-broker:
+    image: apachepinot/pinot:0.12.1
+    command: "StartBroker -zkAddress zookeeper:2181"
+    restart: unless-stopped
+    container_name: "pinot-broker"
+    ports:
+      - "8099:8099"
+    environment:
+      JAVA_OPTS: "-Dplugins.dir=/opt/pinot/plugins -Xms4G -Xmx4G -XX:+UseG1GC -XX:MaxGCPauseMillis=200 -Xloggc:gc-pinot-broker.log"
+    depends_on:
+      - pinot-controller
+    networks:
+      services-network:
+        aliases:
+          - pinot-broker
+  pinot-server:
+    image: apachepinot/pinot:0.12.1
+    command: "StartServer -zkAddress zookeeper:2181"
+    restart: unless-stopped
+    container_name: "pinot-server"
+    ports:
+      - "8098:8098"
+    environment:
+      JAVA_OPTS: "-Dplugins.dir=/opt/pinot/plugins -Xms4G -Xmx16G -XX:+UseG1GC -XX:MaxGCPauseMillis=200 -Xloggc:gc-pinot-server.log"
+    depends_on:
+      - pinot-broker
+    networks:
+      services-network:
+        aliases:
+          - pinot-server
+
+  pinot-admin:
+    image: apachepinot/pinot:latest
+    container_name: pinot-admin
+    depends_on:
+      - pinot-controller
+      - pinot-broker
+      - pinot-server
+    entrypoint: ["/bin/sh", "-c", "cp /schema/pinot/create_pinot_table.sh /opt/pinot/create_pinot_table.sh && chmod +x /opt/pinot/create_pinot_table.sh && /opt/pinot/create_pinot_table.sh"]
+    volumes:
+      - ../../schema:/schema  # Ensure the schema files and script are available in the container
+    networks:
+      services-network:
+        aliases:
+          - pinot-server
+
+  integration-test-cassandra-pinot:
+    build:
+      context: ../../
+      dockerfile: ./docker/buildkite/Dockerfile
+    command:
+      - /bin/sh
```

Comment:
This is some truly weird line endings, I think the `>` can be used to collapse the whole thing. It doesn't matter overly much, but it is confusing